src/modules/selects/TagSelect/TagSelect.scss

```diff
@@ -1,11 +1,16 @@
 @import 'index.scss';
 
 .j-tags-select {
+    .Select-multi-value-wrapper {
+        display: flex;
+        flex-wrap: wrap;
+        margin-bottom: ceil($hf-gutter/2);
+    }
     .Select-value {
         background-color: $hellofresh-color;
         border: none;
         border-radius: $hf-border-radius;
-        margin-top: 0;
+        margin-top: ceil($hf-gutter/2);
         vertical-align: middle;
```

Comment:
why keep the vertical align middle after make it flex?

Reply:
@Vxplus thank you!!! 👍

Reply:
1dc8524422c66d1a55bbcce61e3f5f71def0219f